nms/scripts/createOrganization.ts

```diff
@@ -58,12 +58,10 @@ async function createOrUpdateOrganization(
   organizationObject: OrganizationObject,
 ) {
   const organization = await Organization.findOne({
-    where: {
```

Comment:
This worked and still does although it is semantically somewhat incorrect. The doc are unfortunately really vage here. But with the upgrade the types definitions got better and this is now forbidden.